src/Functions/FunctionBase64Conversion.h

```diff
@@ -50,8 +65,17 @@ struct Base64Decode
 
     static size_t perform(const std::span<const UInt8> src, UInt8 * dst)
     {
+        int rc;
         size_t outlen = 0;
-        int rc = base64_decode(reinterpret_cast<const char *>(src.data()), src.size(), reinterpret_cast<char *>(dst), &outlen, 0);
+        if constexpr (variant == Base64Variant::Url)
+        {
+            auto src_padded = preprocessBase64Url(src);
```

Comment:
`std::vector<UInt8> src_padded` (the type is not immediately clear from the right side)